core/modules/layout/plugins/access/date_time_layout_access.inc

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * @file
+ * Plugin to provide access control/visibility based on date and time.
+ */
+class DateTimeLayoutAccess extends LayoutAccessNegatable {
+
+  /**
+   * {@inheritdoc}
+   */
+  function summary() {
+    $from = format_date(strtotime($this->settings['start_time']),'medium');
+    $to = format_date(strtotime($this->settings['end_time']),'medium');
+    $range = t('from ') . $from . t(' to ') . $to; 
+    if ($this->settings['negate']) {
+      return t('Date is NOT: @range', array('@range'  => $range));
```

Comment:
:+1:  Was just looking at this yesterday and thought that is what I should have done. But I had to reinstall windows last night :cry: